module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaGroupOwner.md

```diff
@@ -8,35 +18,41 @@ schema: 2.0.0
 # Add-EntraBetaGroupOwner
 
 ## Synopsis
+
 Adds an owner to a group.
 
 ## Syntax
 
-```
-Add-EntraBetaGroupOwner -ObjectId <String> -RefObjectId <String> [<CommonParameters>]
+```powershell
+Add-EntraBetaGroupOwner 
+ -ObjectId <String>
+ -RefObjectId <String>
+ [<CommonParameters>]
 ```
 
 ## Description
-The Add-EntraBetaGroupOwner cmdlet adds an owner to an Azure Active Directory group.
+
+The `Add-EntraBetaGroupOwner` cmdlet adds an owner to a Microsoft Entra ID group. Specify the `ObjectId` and `RefObjectId` parameters to add an owner to a group.
```

Comment:
You could explain what `ObjectId` and `RefObjectId` are similar to what was done [here](https://github.com/microsoftgraph/entra-powershell/blob/main/module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraDeviceRegisteredOwner.md#example-1)

Reply:
File updated.